editor/editor_plugin.cpp

```diff
@@ -450,7 +450,7 @@ void EditorPlugin::_bind_methods() {
 	gizmo.return_val.hint = PROPERTY_HINT_RESOURCE_TYPE;
 	gizmo.return_val.hint_string = "EditorSpatialGizmo";
 	ClassDB::add_virtual_method(get_class_static(), gizmo);
-	ClassDB::add_virtual_method(get_class_static(), MethodInfo(Variant::STRING, "get_name"));
+	ClassDB::add_virtual_method(get_class_static(), MethodInfo(Variant::STRING, "_get_name"));
```

Comment:
[Running a search for `add_virtual_method`](https://github.com/godotengine/godot/search?q=add_virtual_method&type=Code&utf8=%E2%9C%93), we can see that no methods starting with an underscore are registered using it.
Looking at the results, I'd suggest using `get_visible_name()`, as it is done for import plugins.